ink-wrapper/src/extensions.rs

```diff
@@ -41,6 +44,10 @@ impl TypeExtensions for Type<PortableForm> {
         self.path().segments().len() == 1
     }
 
+    fn is_custom(&self) -> bool {
+        !self.is_primitive() && !self.is_ink() && !self.is_builtin()
```

Comment:
Can't find it now but I thought that I had seen this mix of conditions somewhere here already...

Reply:
You probably mean here:

```rust
            $(if !typ.ty().is_primitive() && !typ.ty().is_ink() && !typ.ty().is_builtin() {
```

good catch!